Source/Core/Common/SmallVector.h

```diff
@@ -79,11 +83,11 @@ class SmallVector final
     return data()[i];
   }
 
-  auto data() { return std::launder(reinterpret_cast<value_type*>(m_array.data())); }
+  auto data() { return m_array.data()->Ptr(); }
   auto begin() { return data(); }
   auto end() { return data() + m_size; }
 
-  auto data() const { return std::launder(reinterpret_cast<const value_type*>(m_array.data())); }
+  auto data() const { return m_array.data()->Ptr(); }
```

Comment:
(note: the potential call to a not-constructed value happens here for a for-each loop when `m_size == 0` )